torch/__init__.py

```diff
@@ -892,6 +892,12 @@ def compiled_with_cxx11_abi():
 from torch._ops import ops
 from torch._classes import classes
 
+# decompositions_for_jvp depends on torch.ops
+# Registers decompositions for jvp to the jit registry
+if os.environ.get("PYTORCH_JIT", "1") == "1":
+    from torch._decomp import decompositions_for_jvp
+    del decompositions_for_jvp
```

Comment:
Hey @eellison, any idea when `PYTORCH_JIT=0` is used? Should we be worried about users running into this and randomly having less coverage?

Reply:
This should probably be OK now:
- This shouldn't be a regression
- I've updated the error message now with `Note that forward AD support for some operators require JIT to be enabled. If the environment var PYTORCH_JIT=0 is set, some operators may no longer be used with forward AD`.

Reply:
I think it's extremely unlikely that will be used - tbh I forgot we have it and I don't know that we have a CI Job for it.

Reply:
Ah ok, thanks - there was a test that tries to import torch with PYTORCH_JIT=0 which is why I added this. I don't know if the message is super necessary anymore, but will keep for now.